util/metrics/metrics.go

```diff
@@ -88,4 +100,34 @@ var (
 			Name:      GetAlgodRawBlockTimeName,
 			Help:      "Total response time from Algod's raw block endpoint in seconds.",
 		})
+
+	ImporterTimeSeconds = prometheus.NewSummary(
+		prometheus.SummaryOpts{
+			Subsystem: "indexer_daemon",
```

Comment:
Are these specific to the indexer or will they also be available in any conduit pipeline?

Reply:
they will be available to any conduit pipeline. I've thought about setting subsystem to "conduit" but thought it would be confusing to have both indexer_daemon and conduit subsystems showing up in the metrics.